common/browser_process_meta.go

```diff
@@ -0,0 +1,72 @@
+package common
+
+import (
+	"errors"
+	"os"
+
+	"github.com/grafana/xk6-browser/storage"
+)
+
+// ErrUnknownRemoteBrowserProcessPid indicates that the pid for the browser process is unknown
+// as the browser is being executed in a remote machine.
+var ErrUnknownRemoteBrowserProcessPid = errors.New("browser process is remote and pid unknown")
```

Comment:
Do we need to export these types as they are internal to the `common` package?

Reply:
So you mean all the types defined for `BrowserProcessMeta`? It's not necessary to have them exported, I did it to be kind of consistent with other implementations, for example `BrowserProcess` is also internal to `common` pkg. But I guess in this case it's even more internal, so it probably makes sense to not export this.
I will change it.

Reply:
Changed, see [here](https://github.com/grafana/xk6-browser/blob/feat/17-browsertype-connect/common/browser_process_meta.go).